server/src/services/tooljet_db.service.ts

```diff
@@ -430,4 +435,30 @@ export class TooljetDbService {
 
     throw new NotFoundException('Some tables are not found');
   }
+
+  private async editColumn(organizationId: string, params) {
+    const { table_name: tableName, column } = params;
+    const internalTable = await this.manager.findOne(InternalTable, {
+      where: { organizationId, tableName },
+    });
+
+    if (!internalTable) throw new NotFoundException('Internal table not found: ' + tableName);
+    let query = '';
+
+    if (column?.column_default)
+      query += `ALTER TABLE "${internalTable.id}" ALTER COLUMN ${
+        column.column_name
+      } SET DEFAULT ${this.addQuotesIfString(column['column_default'])};`;
+    if ('isNotNull' in column) {
+      column.isNotNull
+        ? (query += `ALTER TABLE "${internalTable.id}" ALTER COLUMN ${column.column_name} SET NOT NULL;`)
+        : (query += `ALTER TABLE "${internalTable.id}" ALTER COLUMN ${column.column_name} DROP NOT NULL;`);
+    }
+
+    if (column?.column_name && column?.new_column_name)
+      query += `ALTER TABLE "${internalTable.id}" RENAME COLUMN ${column.column_name} TO ${column.new_column_name};`;
+    const result = await this.tooljetDbManager.query(query);
```

Comment:
Do we need to explicitly handle the case wherein there maybe existing column values which has null values and we are then editing the column to have not null constraint?